tsconfig.json

```diff
@@ -4,17 +4,21 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "isolatedModules": true,
-    "jsx": "react",
+    "jsx": "react-native",
```

Comment:
should we also change this in the example?

Reply:
Yep, will follow-up